src/hooks/useSlider.js

```diff
@@ -3,26 +3,26 @@ import { useSpring } from 'react-spring'
 import { useDrag } from 'react-use-gesture'
 import { clamp } from '../helpers'
 
-export default function useSlider({ slides }) {
+export default function useSlider({ initialSlide, slides }) {
+  const slideIndex = clamp(initialSlide, 0, slides.length - 1)
```

Comment:
I'm starting to lean away from clamping the `initialSlide` value 😬

Given the user supplies both the `slides` array and `initialSlide` value, I think we should leave error handling/validation up to them.